CONTRIBUTING.md

```diff
@@ -52,6 +52,28 @@ these, open an issue in corresponding repository. If in doubt, ask in
 the IRC channel, or try in any one: if needed you will be redirected to
 the right repository. If you are proposing code, open a pull request.
 
+## DCO and Sign-Off for contributions
+
+The [CHAOSS Charter](https://github.com/chaoss/governance/blob/master/project-charter.md) requires that contributions
+are accompanied by a [Developer Certificate of Origin](http://developercertificate.org) sign-off.
+For ensuring it, a bot checks all incoming commits.
+
+For users of the git command line interface, a sign-off is accomplished with the `-s` as part of the commit command: 
+
+```
+git commit -s -m 'This is a commit message'
+```
+
+For users of the GitHub interface (using the "edit" button on any file, and producing a commit from it),
```

Comment:
I think we don't allow this kind of commits. In that case we should remove it from this description.

Reply:
Do you think we don't allow them? In general, it is difficult to tell commits done in the command line from commits done this way (except if you look at the name of the branch). Are we forbidding this in some repos? (certainly, we're not forbidding this in the tutorial, but we could have a separate contributing file for it).

Reply:
(btw, this one is one of those.... ;-) )

Reply:
I thought nobody was using it and we didn't accept commits with this method. If you think is useful, go ahead with this PR.